testing/test_cluster.py

```diff
@@ -56,6 +57,20 @@ def simple_recovery(self, kill_command):
         value = self.rc_client.read(self.table, 'testKey')
         expect(value).equals(('testValue', 1))
 
+    def test_zookeeper_read(self):
+        self.make_cluster(num_nodes=4)
+        self.createTestValue()
+        zk_client = ctu.get_zookeeper_client(self.ensemble)
+
+        # Read the ZooKeeper entry for the table and make sure it looks sane.
+        # This mostly tests our ability to read from ZooKeeper and parse the
+        # GRPC contents correctly.
+        table_data = zk_client.get('/ramcloud/main/tables/test')[0]
```

Comment:
I'm not entirely sure why you're using '/ramcloud/main/tables/test' for table_data. I'm sure there's a good reason. Can I ask you to clarify this in the comment?